.gitignore

```diff
@@ -4,6 +4,7 @@
 .env
 .DS_Store
 venv/
+.venv/
```

Comment:
The `.venv/` entry is duplicated. Consider removing one of these entries to avoid redundancy.